pkg/pool-aclamm/contracts/lib/AclAmmMath.sol

```diff
@@ -112,6 +126,20 @@ library AclAmmMath {
             }
 
             changed = true;
+        } else if (isInRange && currentTime > sqrtQ0State.startTime && currentTime <= sqrtQ0State.endTime) {
+            uint256 rACenter = lastVirtualBalances[0].mulDown(sqrtQ0State.startSqrtQ0 - FixedPoint.ONE);
+            uint256 rBCenter = lastVirtualBalances[1].mulDown(sqrtQ0State.startSqrtQ0 - FixedPoint.ONE);
+
+            uint256 currentSqrtQ0 = calculateSqrtQ0(
+                currentTime,
+                sqrtQ0State.startSqrtQ0,
+                sqrtQ0State.endSqrtQ0,
+                sqrtQ0State.startTime,
+                sqrtQ0State.endTime
+            );
+
+            virtualBalances[0] = rACenter.divDown(currentSqrtQ0 - FixedPoint.ONE);
+            virtualBalances[1] = rBCenter.divDown(currentSqrtQ0 - FixedPoint.ONE);
```

Comment:
As I said in the call, we probably will need to review this math, but I'll do that and then we can reimplement in another task. Right now it's fine as is.